protocols/v1/src/methods/server_to_client.rs

```diff
@@ -501,29 +501,39 @@ impl TryFrom<&Response> for Configure {
         // Deserialize version-rolling response.
         // Composed by 3 required fields
         let version_rolling: Option<VersionRollingParams>;
-        if version_rolling_.is_some()
-            && version_rolling_mask.is_some()
-            && version_rolling_min_bit_count.is_some()
+        if version_rolling_.is_some() && version_rolling_mask.is_some()
         {
             let vr: bool = version_rolling_
                 .unwrap()
                 .as_bool()
                 .ok_or_else(|| ParsingMethodError::UnexpectedObjectParams(params.clone()))?;
+            
             let version_rolling_mask: HexU32Be = version_rolling_mask
                 .unwrap()
                 .as_str()
                 .ok_or_else(|| ParsingMethodError::UnexpectedObjectParams(params.clone()))?
                 .try_into()?;
-            let version_rolling_min_bit_count: HexU32Be = version_rolling_min_bit_count
-                .unwrap()
-                .as_str()
-                .ok_or_else(|| ParsingMethodError::UnexpectedObjectParams(params.clone()))?
-                .try_into()?;
+
+            // version-rolling.min-bit-count is often not returned by stratum servers,
+            // but min-bit-count should be taken into consideration in the returned mask
+            let version_rolling_min_bit_count: HexU32Be = match version_rolling_min_bit_count {
```

Comment:
Would love it if you could add tests to cover these changes!

Reply:
Added `configure_response_parsing_all_fields` and `configure_response_parsing_no_vr_min_bit_count` in `protocols/v1/src/methods/server_to_client.rs` which should test these changes.